applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -298,6 +281,19 @@ def FinalizeSolutionStep(self):
             self._GetSolutionStrategy().FinalizeSolutionStep()
             self._GetAccelerationLimitationUtility().Execute()
 
+    def __LevelSet_Convection(self):
```

Comment:
```suggestion
    def __LevelSetConvection(self):
```
CamelCase for methods (style guide)

Reply:
Done.

Reply:
I'd also rename it to something like `__ExecuteLevelSetConvection()` or `__PerformLevelSetConvection()` as the current name could be easily misunderstood by a member variable.

Reply:
Done as you suggested.